openjdk/excludes/ProblemList_openjdk16-openj9.txt

```diff
@@ -391,7 +391,6 @@ vm/verifier/VerifyProtectedConstructor.java	https://github.com/eclipse-openj9/op
 ############################################################################
 
 # jdk_foreign
-java/foreign/TestHandshake.java https://github.com/eclipse-openj9/openj9/issues/13211 generic-all
```

Comment:
I don't think we should be modifying versions which are out of support and not getting any updates. 
i.e. 16, 18, 19, 20

Reply:
Updated.